commanderbot_ext/faq/faq_store.py

```diff
@@ -76,11 +75,13 @@ async def add_guild_faq(self, guild: Guild, faq_entry: FaqEntry):
         guild_data.entries[faq_entry.name] = faq_entry
         await self.dirty()
 
-    async def remove_guild_faq(self, guild: Guild, faq_name: str) -> Optional[FaqEntry]:
+    async def remove_guild_faq(self, guild: Guild, faq: FaqEntry) -> str:
```

Comment:
This should still be `Optional[str]` because it's in a conditional block, even if an `FaqEntry` being passed in logistically implies that `guild_data` *ought* to exist. Either that, or remove the conditional block and `assert` that `guild_data` exists after attempting to get it so that either a `str` is returned or an exception is thrown.

Reply:
I'll just assert guild data exists; pretty sure it must, since the faq exists